apps/nextjs-website/src/components/EditorialSwitch.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import React from 'react';
+import MarkdownRenderer from './MarkdownRenderer';
+import { EditorialSwitch as EditorialSwitchRC } from '@react-components/components';
+import { EditorialSwitchProps } from '@react-components/types';
+import { EditorialSwitchSection } from '@/lib/fetch/types/PageSection';
+
+const makeEditorialSwitchProps = ({
+  toptitle,
+  topsubtitle,
+  sections,
+  ...rest
+}: EditorialSwitchSection): EditorialSwitchProps => ({
+  toptitle,
+  ...(topsubtitle && { subtitle: MarkdownRenderer({ markdown: topsubtitle }) }),
+  sections: sections.map(({ button, content }) => ({
+    button,
+    content: {
+      ...content,
+      body: MarkdownRenderer({ markdown: content.body, variant: 'body2' }),
+    },
+  })),
+  ...rest,
+});
```

Comment:
```suggestion
const makeEditorialSwitchProps = ({
  sections,
  ...rest
}: EditorialSwitchSection): EditorialSwitchProps => ({
  sections: sections.map(({ button, content }) => ({
    button,
    content: {
      ...content,
      body: MarkdownRenderer({ markdown: content.body, variant: 'body2' }),
    },
  })),
  ...rest,
});
```
Removed `toptitle` because unchanged.
Removed `topsubtitle` because there is no `subtitle` field and `topsubtitle` is a string anyway, not a JSX.Element as returned from `MarkdownRenderer`.